### DISCUSSION TITLE
Deleting multiple files in a repository.

### DISCUSSION THREAD

**Discussion by LuminoByte** [community/community#51992] [New to GitHub]
Hey, is there a way to delete multiple files in a repository? Instead of having to go through every, single, one of the files i want to delete. Also so it can be in a single commit.

I look forward to hearing from any of you,
Miel

**Comment by airtower-luna** [comment]
Not in the web UI, if that's what you mean.

If you clone the repository to your computer, you can `git rm` as many files as you want and then commit (and push, probably). If you're new to using Git, check the [Git Book](https://git-scm.com/book/en/v2) website.
